README.md

```diff
@@ -64,9 +64,36 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Run commitlint
+        id: commitlintrun
         uses: opensource-nepal/commitlint@v0.2.1
+        with:
+          fail_on_error: false
```

Comment:
issue: This is a basic usage of the commitlint actions. Let's skip adding fail_on_error here. Maybe we could include a new example instead.

Reply:
Done